fetch.bs

```diff
@@ -7440,27 +7439,100 @@ fetch("https://www.example.com/")
 
 <h3 id=websocket-connections>Connections</h3>
 
+<p>A user agent has an associated <dfn>connecting WebSocket connections</dfn>, a <a for=/>map</a> of
+(<a for=/>network partition key</a>, <a for=/>host</a>, <a for=url>port</a>) to a
+<a for=/>set</a> of unique identifiers.
+
+<p>A user agent has an associated <dfn>total pending proxy WebSocket connections</dfn>, a
+<a for=/>map</a> of <a for=/>network partition key</a> to an integer, starting at 0.
+
 <p>To <dfn id=concept-websocket-connection-obtain>obtain a WebSocket connection</dfn>, given a
-<var>url</var>, run these steps:
+<a>network partition key</a> <var>key</var>, <a for=/>origin</a> <var>origin</var>, and boolean
+<var>credentials</var>, run these steps:
 
 <ol>
- <li><p>Let <var ignore>host</var> be <var>url</var>'s <a for=url>host</a>.
+ <li><p>Let <var>proxies</var> be the result of performing proxy resolution for <var>url</var> in an
+ <a>implementation-defined</a> manner. If a proxy preference order has not been explicitly
+ configured then socks proxies should be preferred to https proxies which should be preferred to
+ http proxies.
+
+ <!-- Since this depends on the url it probably needs to be hoisted out of this algorithm. -->
+
+ <p class=note>The scheme of the URL used in proxy autoconfiguration scripts has been converted from
+ "<code>ws</code>" or "<code>wss</code>" to "<code>http</code>" or "<code>https</code>". This
```

Comment:
Does this mean that when the user agent invokes `FindProxyForURL` this is the case or something else? (Just want to make sure I understand.)